autoapi/extension.py

```diff
@@ -124,4 +126,5 @@ def setup(app):
     app.add_config_value('autoapi_keep_files', False, 'html')
     app.add_config_value('autoapi_add_toctree_entry', True, 'html')
     app.add_config_value('autoapi_template_dir', [], 'html')
+    app.add_config_value('autoapi_url_root', URL_ROOT, 'html')
```

Comment:
It's relative in every place: http://sphinx-autoapi.readthedocs.org/en/latest/config.html#confval-autoapi_root & https://github.com/rtfd/sphinx-autoapi/blob/master/autoapi/extension.py#L119

It gets normalized just above the code you edited: https://github.com/rtfd/sphinx-autoapi/blob/master/autoapi/extension.py#L46